requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 cvxpy>=1.0.15
 cvxopt>=1.2.3
+qiskit-terra>=0.7.0
```

Comment:
This isn't needed, this requirements file is just for testing purposes. The real project requirements live in setup.py and already includes qiskit-terra: https://github.com/Qiskit/qiskit-ignis/blob/master/setup.py#L11-L15